tests/integration/pilot/analysis/analysis_test.go

```diff
@@ -0,0 +1,79 @@
+// Copyright 2019 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package analysis
+
+import (
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/bookinfo"
+	"istio.io/istio/pkg/test/framework/components/deployment"
+	"istio.io/istio/pkg/test/framework/components/environment/kube"
+	"istio.io/istio/pkg/test/framework/components/namespace"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"strings"
+	"testing"
+)
+
+func TestAnalysisWritesStatus(t *testing.T) {
+	framework.NewTest(t).Run(func(ctx framework.TestContext) {
+		ns := setupModifiedBookinfo(t, ctx)
+		doTest(t, ctx, ns)
+	})
+}
+
+func setupModifiedBookinfo(t *testing.T, ctx framework.TestContext) namespace.Instance {
```

Comment:
@gargnupur mind reviewing for general usage of bookinfo and its gateway.

Reply:
just to be clear, this is still WIP, even after the multiple pilot issue is resolved, I need to mutate bookinfo somehow to trigger an Analyzer message. My plan was to misspell the gateway name, but now that I look at it, these virtualservices don't directly reference gateways...

Reply:
Why do you even need to deploy bookinfo? Can't you just directly run

```
applyconfig(`some-broken-config`)
```

and then check the status? We do not need to deploy any pods?

Reply:
That is true.  I figured it would be easier to use an existing reference application than to spend time authoring and maintaining custom config just for this test.

Reply:
I would agree in general but you are already not doing something standard (invalid config), applying a config is a simple one line, and deploying bookinfo has a runtime cost which doesn't seem to benefit at all here

Reply:
ok, I'll modify my test.